packages/api/src/external/carequality/cq-patient-data.ts

```diff
@@ -1,8 +1,10 @@
 import { BaseDomain, BaseDomainCreate } from "@metriport/core/domain/base-domain";
+import { InboundPatientResource } from "@metriport/ihe-gateway-sdk";
 
 export type CQExternalPatient = {
   patientId: string;
   systemId: string;
+  patientResource?: InboundPatientResource;
```

Comment:
for (1) -- it's a good point. Because we're "additive" for CQ links in the table, simply re-running PD flow for a patient  may not be sufficient to update previously found links...

Needs to be resolved for sure (it should be possible with some gnarly SQL along patient discovery result).

Reply:
Irrelevant: we now use the patient state for building augmented data, which means we don't use this table this way (and we likely don't need to populate retroactively).

Reply:
For (2): https://github.com/metriport/metriport-internal/issues/1841